tests/qft_tests.py

```diff
@@ -7,8 +7,13 @@ def test_qft(self):
         trial_circ = build_circuit('X_1')
         trial_state.apply_circuit(trial_circ)
 
+        # verify transformation
         qft(trial_state, 4)
 
+        # test unitarity
+        qft(trial_state, 3)
+        rev_qft(trial_state, 3)
+
```

Comment:
I would test the qft more extensively. For example, can you do a QFT of a more complicated state and check if the coefficient of the final wave function match some reference values? Same for the inverse. However, I would also keep a test where you start from a state and do qft followed by reverse qft to make sure the final state is the same as the beginning one. We can talk about it if you want.